tests/runtime/ray/test_integration.py

```diff
@@ -37,9 +37,16 @@ def runtime(
     config = configs.RuntimeConfiguration(
         config_name="test-config",
         runtime_name=configs.RuntimeName.RAY_LOCAL,
+        runtime_options={
+            "address": None,
+            "log_to_driver": True,
+            "storage": None,
+            "temp_dir": None,
+            "configure_logging": None,
+        },
```

Comment:
**question**: ray configuration option in tests is a super delicate topic. What change does this diff make?

Reply:
Previously we used those runtime_options is from_runtime_configuration. Now as I removed this function I added its logic to constructor, which is used in our tests. Ill double check all usages of those params if this might ahve any impact on anything

Reply:
**suggestion**: it's very easy to misconfigure Ray, making the tests unreliable. After these changes the options will be defined in multiple places, which increases the risk of silent errors. Could you use a single place where we define the runtime options for Ray config used in tests (SSOT)?

Reply:
Good point. Changed all definitions of this config to point out to our global config we use when user uses "local"